pkg/webhook/patch_test.go

```diff
@@ -1108,6 +1108,98 @@ func TestPatchSparkPod_HostNetwork(t *testing.T) {
 	}
 }
 
+func TestPatchSparkPod_Env(t *testing.T) {
+	drvEnvKey := "TEST_DRV_ENV_VAR_KEY"
```

Comment:
Please use more than one env vars both the driver and executor pods.

Reply:
Done